lib/peanut_labs.rb

```diff
@@ -6,5 +6,5 @@
 require 'peanut_labs/direct_link'
 
 module PeanutLabs
```

Comment:
Not sure if it goes against good code conventions, but I'd inline module definition in this case